toolkit/analysis_and_visualization/classification/assets/styles/style.css

```diff
@@ -0,0 +1,410 @@
+  cursor: pointer;
+}
+
+@media only screen and (max-width: 400px) {
+  .modal-content {
+    width: 100%;
+  }
+}
+
+body {
+  font-family: "Lato", sans-serif;
+}
+
+.sidenav {
+  height: 100%;
+  width: 0;
+  position: fixed;
+  z-index: 7;
+  top: 0;
+  left: 0;
+  background-color: #111;
+  overflow-x: hidden;
+  transition: 0.5s;
+  padding-top: 60px;
+}
+
+.sidenav a {
+  padding: 8px 8px 8px 32px;
+  text-decoration: none;
+  font-size: 25px;
+  color: #818181;
+  display: block;
+  transition: 0.3s;
+}
+
+.sidenav a:hover {
+  color: #f1f1f1;
+}
+
+.sidenav .closebtn {
+  position: absolute;
+  top: 0;
+  right: 25px;
+  font-size: 36px;
+  margin-left: 50px;
+}
+
+#main {
+  transition: margin-left 0.5s;
+  padding: 16px;
+}
+
+@media screen and (max-height: 450px) {
+  .sidenav {
+    padding-top: 15px;
+  }
+
+  .sidenav a {
+    font-size: 18px;
+  }
+}
+
+body {
+  margin: 0;
+}
+
+.navbar {
+  overflow: hidden;
+  background-color: #333;
+  position: fixed;
+  z-index: 6;
+  top: 0;
+  width: 100%;
+}
+
+.navbar a {
+  float: left;
+  display: block;
+  color: #f2f2f2;
+  text-align: center;
+  padding: 14px 16px;
+  text-decoration: none;
+  font-size: 17px;
+}
+
+.navbar a:hover {
+  background: #ddd;
+  color: black;
+}
+
+.main {
+  padding: 16px;
+  margin-top: 30px;
+}
+
+select {
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  text-indent: 0;
+  text-overflow: '';
+  color: maroon;
+}
+
+.tooltip {
+  position: relative;
+  display: inline-block;
+}
+
+.tooltip .tooltiptext {
+  visibility: hidden;
+  width: 150px;
+  background-color: black;
+  color: gold;
+  text-align: center;
+  border-radius: 6px;
+  padding: 5px;
+  position: absolute;
+  z-index: 3;
+}
+
+.tooltip:hover .tooltiptext {
+  visibility: visible;
+}
+
+.footer {
+  position: fixed;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  background-color: #333;
+  color: white;
+  text-align: center;
+}
+
+#mySideNav .nav-item {
+  font-size: large;
+}
+
+.header {
+  color: dodgerblue;
+  font-size: 2em;
+  text-align: center;
+}
+
+.center {
+  text-align: center;
+  margin: auto;
+}
+
+.marron-header th {
+  color: maroon;
+}
+
+#top-5-match-table th {
+  color: maroon;
+  font-weight: bold;
+  text-align: center;
+}
+
+#image-results-table td {
+  font-size: small;
+}
+
+#image-results-table th {
+  color: maroon;
+  font-weight: bold;
+  text-align: center;
+  font-size: small;
+}
+
+#label-summary-table th {
+  color: maroon;
+  font-weight: bold;
+  text-align: center;
+}
+
+#label-summary-table th .blue {
+  color: blue;
+}
+
+#label-summary-table th .black {
+  color: black;
+}
+
+#label-summary-table tbody td {
+  font-size: small;
+  font-weight: bold;
+}
+
+.left-align {
+  text-align: left;
+  font-size: x-small !important;
+}
+
+.color-blue {
+  color: blue;
+}
+
+.color-green {
+  color: green;
+}
+
+.color-red {
+  color: red;
+}
+
+.color-black {
+  color: black;
+}
+
+.highlight-row {
+  background-color: #d5f5e3 !important;
```

Comment:
Codacy found an issue: [Use of !important](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6397592)